### PR TITLE
Adding git-aggregate troubleshooting to README.md

### PR DESCRIPTION
**Pull request by kneckinator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,32 @@ invoke --list
...
+
+The easiest way is to start over by removing the `src` folder and resetting the
+`openspp-docker` repository.
+
+```bash
+rm -rf odoo/custom/src
+git reset --hard
+```
+
+⚠️ Check your branch before resetting the repository. You might lose some work. Stash
```
Comment (jeremi) [inline README.md:167]: if you do `rm -rf odoo/custom/src` stashing will not be enough to avoid loosing anything :)

**Review by jeremi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kneckinator) [reply README.md:167]: I was only thinking of files under control by git, which should be the only files affected. For instance `repos.yaml` and `addons.yaml`.

Comment (kneckinator) [reply README.md:167]: @jeremi I added the full flow in the readme as well to clarify for the dev.

Comment (kneckinator) [reply README.md:167]: I am abandoning this PR as I found a better way to solve it in repos.yaml.

**Pull request closed without merging**